inlong-sort/sort-connectors/kudu/src/main/java/org/apache/inlong/sort/kudu/common/KuduOptions.java

```diff
@@ -0,0 +1,125 @@
+
+    public static final ConfigOption<String> MAX_BUFFER_TIME =
+            ConfigOptions.key("sink.max-buffer-time")
+                    .stringType()
+                    .defaultValue("30s")
+                    .withDescription("The maximum wait time for buffered records in the sink.");
+
+    public static final ConfigOption<String> SINK_KEY_FIELD_NAMES =
+            ConfigOptions.key("sink.key-field-names")
+                    .stringType()
+                    .defaultValue("")
+                    .withDescription("The key fields for updating DB when using upsert sink function.");
+    public static final ConfigOption<Boolean> ENABLE_KEY_FIELD_CHECK =
+            ConfigOptions.key("sink.enable-key-field-check")
+                    .booleanType()
+                    .defaultValue(true)
+                    .withDescription("If true, the check to compare key fields assumed by flink " +
+                            "and key fields provided by user will be performed.");
+
+    public static final ConfigOption<Boolean> SINK_WRITE_WITH_ASYNC_MODE =
+            ConfigOptions.key("sink.write-with-async-mode")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription("Use async write mode for kudu producer if true.");
+
+    public static final ConfigOption<Boolean> SINK_FORCE_WITH_UPSERT_MODE =
+            ConfigOptions.key("sink.write-with-upsert-mode")
+                    .booleanType()
+                    .defaultValue(true)
+                    .withDescription("Force write kudu with upsert mode if true.");
+
+    public static final ConfigOption<Integer> CACHE_QUEUE_MAX_LENGTH =
+            ConfigOptions.key("sink.cache-queue-max-length")
+                    .intType()
+                    .defaultValue(-1)
+                    .withDescription("The maximum queue lengths.");
+    public static final ConfigOption<String> CONNECTOR_TABLE =
+            ConfigOptions.key("connector.table")
+                    .stringType()
+                    .noDefaultValue().withDescription("The name of kudu table.");
+    public static final ConfigOption<String> CONNECTOR_MASTER =
+            ConfigOptions.key("connector.master")
+                    .stringType()
+                    .noDefaultValue().withDescription(" The masters of kudu server.");
+    public static final ConfigOption<Boolean> LAZY_LOAD_SCHEMA =
+            ConfigOptions.key("lazy.load.schema")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription("Whether get metadata from kudu after generating flink job graphic.");
+    public static final ConfigOption<Boolean> KUDU_IGNORE_ALL_CHANGELOG =
+            ConfigOptions.key("ignore.all.changelog")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription("Whether ignore delete/update_before/update_after message.");
```

Comment:
maybe `ignore.all.changelog` change to `sink.ignore.changelog`. keep consistent with other connector